lusidtools/cocoon/cocoon.py

```diff
@@ -661,7 +664,7 @@ def _convert_batch_to_models(
         ):
             sub_holding_keys_row = cocoon.properties.create_property_values(
                 row=row,
-                scope=properties_scope,
+                scope=sub_holding_keys_scope,
```

Comment:
~~Does this possibly introduce a breaking a change? Thinking of a scenario of anyone currently using LPT who is assuming that SHKs scope are the property scopes? In which case should this check for `sub_holdings_keys_scope` being null or empty and revert to `property_scope`? Or are we looking for a clean change and do we need to document somewhere? wiki maybe?~~

Reply:
Ignore above I missed that you already do that below in load_from_data_frame